apps/coinsence-kit/contracts/CoinsenceKit.sol

```diff
@@ -8,6 +8,7 @@ import "@aragon/os/contracts/lib/ens/PublicResolver.sol";
 import "./misc/APMNamehashOpen.sol";
 import "@coinsence/apps-space/contracts/Space.sol";
 import "@coinsence/apps-coin/contracts/Coin.sol";
+import "@aragon/os/contracts/apm/APMNamehash.sol";
```

Comment:
shoudl this go here?

Reply:
ahh sorry will remove it... I was testing locally so I needed that.